modules/nw-route-admission-policy.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * ingress/configure-ingress-operator.adoc
+// * networking/routes/route-configuration.adoc
+
+[id="nw-route-admission-policy_{context}"]
+= Configuring the route admission policy
+
+Administrators and application developers can run applications in multiple namespaces with the same domain name. This is for organizations where multiple teams develop microservices that are exposed on the same host name.
+
+[WARNING]
+====
+Allowing claims across namespaces should only be enabled for clusters with trust between namespaces, otherwise a malicious user could take over a host name. For this reason, the default admission policy disallows host name claims across namespaces.
+====
+
+.Prerequisites
+
+* Cluster administrator privileges.
+
+.Procedure
+
+* Edit the `.spec.routeAdmission` field of the `ingresscontroller` resource variable using the following command:
++
+----
+$ oc edit --namepace openshift-ingress-operator ingresscontrollers/default
+----
++
+Then set `.spec.routeAdmission.namespaceOwnership=InterNamespaceAllowed`.
+//+
+//.Sample Ingress Controller configuration
+//----
```

Comment:
if using `oc edit` command then the field should be updated like:
```
spec:
  routeAdmission:
    namespaceOwnership: InterNamespaceAllowed
```
and you can use `oc patch` command as well:
```
$ oc -n openshift-ingress-operator patch ingresscontroller/default --patch '{"spec":{"routeAdmission":{"namespaceOwnership":"InterNamespaceAllowed"}}}' --type=merge
```

Reply:
Thanks!